File: 1013.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int maiorab(int a, int b);

int main()
{
    int a, b, c, maior;

    cin >> a >> b >> c;

    maior = maiorab(a, b);
    maior = maiorab(maior, c);
    
    

    cout << maior << " eh o maior" << endl;

    return 0;
}

int maiorab(int a, int b)
{
    return (a + b + abs(a - b)) / 2;
}

File: Python/1013.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

def maior_ab(a, b):
    return int((a + b + abs(a - b)) / 2)

A, B, C = input().split()
A = int(A)
B = int(B)
C = int(C)

maior = maior_ab(A, B)
maior = maior_ab(maior, C)

print(f"{maior} eh o maior")
